blotter-template/blade/DataSource/bin/start-jar.sh

```diff
@@ -14,6 +14,12 @@ BLADENAME=@adapterName@
 
 if [ "$1" = "CONFREADER" ]; then
    shift
+   if [[ `uname` == "Linux"  ]]; then
+       machine=`uname -m`
+       "${BINARY_ROOT}/bin/configreader-$machine" "$@" 2> /dev/null
```

Comment:
Use the DFW configuration reader. Note that this means we'll have to check if it exists first. Also, on old DFW's `${PROGROOT}` isn't exported so we'll have to use `${CONFIG_BASE}`.

Reply:
Dom reviewed the change and is happy with it.